EKF/estimator_interface.h

```diff
@@ -54,6 +54,7 @@
 #include <mathlib/mathlib.h>
 
 using namespace estimator;
+using namespace estimator::sensor;
```

Comment:
This is already an issue here, but it's better not to do `using` in a header file, since then anybody who includes it also gets that, without any namespace.

Maybe put these both inside the EstimatorInterface class?

Reply:
I think the estimator interface should be in the estimator namespace. If not, I'll anyway remove the `using` to avoid namespace leaking